nextflow_schema.json

```diff
@@ -327,7 +327,7 @@
                 },
                 "cf_ploidy": {
                     "type": "string",
-                    "default": 2,
+                    "default": "2",
```

Comment:
what happened here?

Reply:
hu must have happened when I ran `nf-core pipelines schema build`

Reply:
can you revert this? this causes issues otherwise. 
```suggestion
                    "default": 2,
```